tools/depends/target/libfstrcmp/Makefile

```diff
@@ -0,0 +1,37 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+# lib name, version
+LIBNAME=fstrcmp
+VERSION=0.7.D001
+SOURCE=$(LIBNAME)-$(VERSION)
+ARCHIVE=$(SOURCE).tar.gz
+
+# configuration settings
+CONFIGURE=./configure --prefix=$(PREFIX) --disable-shared
+
+LIBDYLIB=$(PLATFORM)/.libs/$(LIBNAME)16.a
```

Comment:
this looks like it is copied from libpng

Reply:
aye.